Windows/MainWindow.xaml.cs

```diff
@@ -472,6 +564,18 @@ public partial class MainWindow {
             }
         }
 
+        [CanBeNull]
```

Comment:
IS there some reason this couldn't stay exactly where it was already?

Reply:
No, this must be me failing to rebase properly or something.  Sorry again.